app/src/main/java/com/vero/tasky/MainActivity.kt

```diff
@@ -36,15 +36,23 @@ class MainActivity : ComponentActivity() {
 
         setContent {
             TaskyTheme {
-
+                val localSnackbarHostState = LocalSnackbarHostState.current
                 CompositionLocalProvider(
-                    LocalScaffoldState provides rememberScaffoldState()
+                    LocalSnackbarHostState provides localSnackbarHostState
                 ) {
+                    val errorMessage = viewModel.errorMessage
+                    LaunchedEffect(errorMessage, localSnackbarHostState) {
+                        if (errorMessage == null)
+                            return@LaunchedEffect
+                        else {
+                            localSnackbarHostState.showSnackbar(getString(errorMessage))
+                            viewModel.onMessageErrorSeen()
```

Comment:
Do you really need to "viewModel.onMessageErrorSeen()"?
Pretty sure the Snackbar already does that logic if you do it using Channel (which you do on RegistrationScreen)